### PR TITLE
Enhance Typos Checker docs to run the Demo

### PR DESCRIPTION
**Pull request by dpordomingo**
I fixed some things to run the demo (that was not properly documented)

So this would be the proposal:
https://github.com/irinakhismatullina/style-analyzer/blob/b647a3f5a31e00dc3561bf1f911f4a152fabee0d/lookout/style/typos_checker/README.md

### CODE REVIEW

**Comment by dpordomingo** [comment]
@meyskens @tsolakoua feel free to propose any change ;)

**Comment by irinakhismatullina** [comment]
Everything looks fine to me

File: lookout/style/typos_checker/README.md [hunk 1]
```diff
@@ -1,53 +1,86 @@
...
+# How to run it
 
-## how to reproduce
+You can follow these steps to reproduce the demo, as it was seen during the presentation of the hackathon project.
+
+PR and issues are very welcomed ;)
+
+## Run lookout
+
+You will need to have [Go](https://golang.org/doc/install#install) and [docker-compose](https://docs.docker.com/compose/install/)
```
Comment (tsolakoua) [inline lookout/style/typos_checker/README.md]: suggestion: You will need to have [Go](https://golang.org/doc/install#install) and [docker-compose](https://docs.docker.com/compose/install/) installed in your machine

File: lookout/style/typos_checker/README.md [hunk 2]
```diff
@@ -1,53 +1,86 @@
...
 ### 3. Run lookout server
+
 ```shell
 $ lookoutd serve --github-token <gh-token> --github-user <user> <repository>
 ```
+
 where:
 - `user` is the github user handler that will post the comment,
 - `gh-token` is [the token of the user](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line) that will post the comment,
 - `repository` is the repository url that will be watched for new PRs (it MUST be public)
```
Comment (tsolakoua) [inline lookout/style/typos_checker/README.md]: s/url/URL

Comment (tsolakoua) [reply lookout/style/typos_checker/README.md]: s/(it MUST be public)/and should be public

File: lookout/style/typos_checker/README.md [hunk 3]
```diff
@@ -1,53 +1,86 @@
...
 
-### 4. Change analyzer `config.yml`; exposed port should be `10302`
+## Run Style Analyzer
+
+You will need to have [Python](https://www.python.org/downloads/) (between `3.5` and `3.7`)
```
Comment (tsolakoua) [inline lookout/style/typos_checker/README.md]: suggestion: If I'm not wrong 3.7 is the latest version so something like higher than 3.5 should be fine as well

File: lookout/style/typos_checker/README.md [hunk 4]
```diff
@@ -1,53 +1,86 @@
...
+
+```shell
+$ sudo pip3 install -e .
+```
+
+### 2. Configure and run the analyzer
+
+Change the analyzer [`config.yml`](https://github.com/irinakhismatullina/style-analyzer/blob/feature/typos-analyzer/config.yml) to expose the `10302` port.
+
+And run it from **Style Analyzer** directory:
```
Comment (tsolakoua) [inline lookout/style/typos_checker/README.md]: same here

File: lookout/style/typos_checker/README.md [hunk 5]
```diff
@@ -1,53 +1,86 @@
...
-### 4. Change analyzer `config.yml`; exposed port should be `10302`
+## Run Style Analyzer
+
+You will need to have [Python](https://www.python.org/downloads/) (between `3.5` and `3.7`)
+
+### 1. Install style-analyzer
+
+Download the **Style Analyzer** repsitory, containing the `typos-checker` module, that can be found at [`irinakhismatullina/style-analyzer` at `typos-analyzer` branch](https://github.com/irinakhismatullina/style-analyzer/tree/feature/typos-analyzer)
+
+And install its dependencies, running from its folder:
```
Comment (tsolakoua) [inline lookout/style/typos_checker/README.md]: I don't like a lot "And" at the beginning of sentences as it's not very common, but I think it's still right :)

File: lookout/style/typos_checker/README.md [hunk 6]
```diff
@@ -1,53 +1,86 @@
...
 
-### 5. Install style-analyzer as it can be found at [irinakhismatullina/style-analyzer::typos-analyzer](https://github.com/irinakhismatullina/style-analyzer/tree/feature/typos-analyzer)
 ```shell
 $ python3 -m lookout run lookout.style.typos_checker -c config.yml
 ```
 
-### 6. Open a PR with a Java file, containing some typos in some code comments
+## Test it with an example in GitHub
+
+Open a PR &ndash;into the watched repository&ndash; with a Java file, containing some typos in some code comments, example:
```
Comment (tsolakoua) [inline lookout/style/typos_checker/README.md]: Open a PR &ndash;into the watched repository&ndash; with a Java file, containing some typos in some code comments, for example:
(I added the ```for```)

File: lookout/style/typos_checker/README.md (hunk 2 above)
Comment (dpordomingo) [reply lookout/style/typos_checker/README.md]: I'm using SHOULD/MUST as defined by the [RFC2119](https://www.ietf.org/rfc/rfc2119.txt), so I tried to explain that it is a hard requirement, not a recommendation.
Would you agree on that?

File: lookout/style/typos_checker/README.md (hunk 3 above)
Comment (dpordomingo) [reply lookout/style/typos_checker/README.md]: I just followed @vmarkovtsev recommendations at [style-analyzer/getting_started](https://github.com/src-d/style-analyzer/blame/master/lookout/core/doc/getting_started.md#L8) guide.
He might be not confident about back compatibility on new Python versions. Would you agree on that?

File: lookout/style/typos_checker/README.md (hunk 4 above)
Comment (dpordomingo) [reply lookout/style/typos_checker/README.md]: What about `Next` or `Then`?

File: lookout/style/typos_checker/README.md (hunk 5 above)
Comment (dpordomingo) [reply lookout/style/typos_checker/README.md]: What about `Next` or `Then`?

**Comment by dpordomingo** [comment]
Many thanks for the deep review @tsolakoua !
I had some questions and clarifications; I'll solve those and the accepted ones as soon as possible, and I'll ping you again when it's ready for a next review :D

File: lookout/style/typos_checker/README.md (hunk 2 above)
Comment (tsolakoua) [reply lookout/style/typos_checker/README.md]: Yes, I agree!! I didn't know about it before, it's good to know :)

File: lookout/style/typos_checker/README.md (hunk 3 above)
Comment (tsolakoua) [reply lookout/style/typos_checker/README.md]: Before I add the comment, I checked that the latest version of Python is 3.7.0, so I thought it's not necessary to add the upper limit. However, I don't know how often Python releases a new version... Anyway, we can leave it as Vadim has it :)

File: lookout/style/typos_checker/README.md (hunk 4 above)
Comment (tsolakoua) [reply lookout/style/typos_checker/README.md]: Both are good. If you add Next it needs a comma ```Next,```  - I just checked it on Grammarly

**Comment by tsolakoua** [comment]
Thanks @dpordomingo  :)))

File: lookout/style/typos_checker/README.md (hunk 4 above)
Comment (dpordomingo) [reply lookout/style/typos_checker/README.md]: thanks!

**Comment by dpordomingo** [comment]
Ready for a new pass, thanks @tsolakoua

File: lookout/style/typos_checker/README.md [hunk 7]
```diff
@@ -26,7 +26,7 @@ $ go get github.com/src-d/lookout/...
 $ docker-compose up bblfsh postgres
 ```
 
-and then initialize **lookout** database
+Next, then initialize **lookout** database
```
Comment (tsolakoua) [inline lookout/style/typos_checker/README.md]: Next, initialize lookout database 
or Then, initialize lookout database

**Review by tsolakoua — APPROVED** [review]
Apart from the comment I left, everything looks fine. Thanks @dpordomingo

File: lookout/style/typos_checker/README.md (hunk 7 above)
Comment (dpordomingo) [reply lookout/style/typos_checker/README.md]: thanks, I will fix it

**Pull request merged**